src/coreclr/src/debug/dbgutil/CMakeLists.txt

```diff
@@ -5,8 +5,18 @@ endif(CLR_CMAKE_HOST_WIN32)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+include_directories(${CLR_DIR}/src/inc/llvm)
+
+add_definitions(-DPAL_STDCPP_COMPAT)
+
 set(DBGUTIL_SOURCES
     dbgutil.cpp
 )
 
+if(CLR_CMAKE_HOST_LINUX)
```

Comment:
Ultimately this will need to be TARGET_LINUX for the cross OS DAC.  I can fix later when I enable this for the cross OS DAC.